test/DotNetEnvTraverse.Tests/TraverseTests.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Linq;
+using DotNetEnv;

```

Comment:
Reverted all syntactic changes in all the tests, but there is truly one single breaking change, though it's only a required using. All the others are still working as expected.

Moving all the methods to an extension requires the using to be explicitly present.
I do not see a way araound this, except for moving the methods back.

I think even with moving the methods back into the LoadOptions the change is still leading to better readability, so your decision:
Should I move it or is this change acceptable?

Reply:
So, this is because of the previous change you made that created and added the `Extensions` using instead of just the base `DotNetEnv`, correct?

Reply:
Yes, because in the TraverseTests every statement explicitly uses `DotNetEnv.XXX` instead of `using` it.
The extension cannot be called in this way (except if you call it "method-style" instead of "extension-style" of course) and that's why the using is needed with the extension.